modules/NodeColorManager/src/main/nbm/manifest.mf

```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+OpenIDE-Module-Name: Node Color Manager
+OpenIDE-Module-Short-Description: allow you share node colors among multi graph
```

Comment:
For the sake of presentation convention can you use a capital letter here ex: "Share node colors...".